src/libraries/Microsoft.Win32.Registry.AccessControl/Directory.Build.props

```diff
@@ -2,6 +2,6 @@
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
     <StrongNameKeyId>Microsoft</StrongNameKeyId>
-    <IsWindowsSpecific>true</IsWindowsSpecific>
+    <SupportedOSPlatforms>windows</SupportedOSPlatforms>
```

Comment:
I thought `SupportedOSPlatforms` was an Item and not a property?

Reply:
There are 2 with similar naming:

- `<SupportedPlatform>`, which is an item, and it tells the analyzer which platforms we care to see warnings about, and it's part of the SDK product
- `<SupportedOSPlatform>`, which is this property, and it results in assembly-level `[SupportedOSPlatform]` attributes, and it's part of the runtime repo only

These names are so similar they're surely easy to get mixed up.  Perhaps a better name for the runtime repo properties that produce the assembly-level attributes would be:

- `<AssemblySupportedOSPlatformAttributes>`
- `<AssemblyUnsupportedOSPlatformAttributes>`